### PR TITLE
D docpage

### PR DESCRIPTION
**Pull request by Ayushi20-19**
Created Documentation page, create a sidebar, and add Installation section, fix the navbar also.
image

### CODE REVIEW

**Review by aamir74 — APPROVED** [review]
Good to go

File: doc.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="navbar">
+      <h5 class="logo">BlossomUI</h5>
+      <!-- <img class="logo" src="images/logo.png" alt="logo of website  BlossomUI" /> -->
```
Comment (VishalPatil18) [inline doc.html:20]: Try avoiding commented Code in the PR.

File: style.css [hunk 2]
```diff
@@ -32,6 +31,11 @@
   height: 15vh;
   display: flex;
   align-items: center;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  position: fixed;
+  background-color: var(--white-color);
```
Comment (VishalPatil18) [inline style.css:38]: As there are multiple properties here you could sort them alphabetically which helps in debugging and code maintainability.

**Review by ruchiragoswami — APPROVED** [review]
Looks good Ayushi :)

File: doc.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          <p>
+            Copy and paste the code in the head tag of your html document. So
+            that you can style your components by just adding class names to
+            your html elements.
+          </p>
+        </div>
+
+        <div class="code_link">
+          <code>
+            <iframe
```
Comment (VishalPatil18) [inline doc.html:73]: Here, the height of this container can be adjusted as it's making it scroll, which is not a very good UI.
Screenshot 2022-02-01 at 21 46 36

**Review by VishalPatil18 — APPROVED** [review]
Just a couple of suggestions were added. Apart from that everything looks good!

File: doc.html (hunk 3 above)
Comment (Ayushi20-19) [reply doc.html:73]: will resolve this in next pr

File: style.css (hunk 2 above)
Comment (Ayushi20-19) [reply style.css:38]: thanks will keep in mind

File: doc.html (hunk 1 above)
Comment (Ayushi20-19) [reply doc.html:20]: this one is purposely as i have to add logo instead of h5, but i can remove it for now

**Pull request merged**